bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/PulseaudioClient.java

```diff
@@ -145,10 +145,29 @@ public boolean isConnected() {
         return client != null ? client.isConnected() : false;
     }
 
+    public void tryUpdate() {
+        var updateRetries = 3;
+        while (updateRetries > 0) {
+            try {
+                update();
+            } catch (IOException e) {
+                if (updateRetries == 1) {
+                    logger.warn("IOException updating state: {}", e.getMessage());
+                }
+                updateRetries--;
+                try {
+                    Thread.sleep(0);
+                } catch (InterruptedException ignored) {
+                }
+            }
+            break;
```

Comment:
With a break at this place, a retry will never happen. 
So if you tell me that the new code solved many problems, I doubt it is due any retry.
WDYT?

Reply:
I don't know what to think, 🤔 I've tested the changes and see the duplicate problem solved. I don't know if I break the mechanism before the commit or it was fixed by other reason. I will test it again this afternoon and try to understand what happened.

Reply:
I think I understood what is happening, the modules info was loaded before, so my changes only catch the error on the update call, and let the process continue. So it was normal I stop seeing errors as they were all ignored. 
And everything works as the modules are loaded always before turn the things online.

That makes me realise that I can change the place were the update is taking place as it's not really required to reload the modules before looking for a valid module, just after load a new one.